coffee.TODO

```diff
@@ -0,0 +1,260 @@
+28 ./app/views/questions/render_question_parameters.coffee
+28 ./app/views/remarks/cancel_remark_basics.coffee
+28 ./app/views/submissions/create.coffee
+29 ./app/views/answers/create.coffee
+29 ./app/views/quizzes/edit_vertex_targets.coffee
+
+30 ./app/views/assignments/create.coffee
+30 ./app/views/quizzes/update_branching.coffee
+30 ./app/views/remarks/reassign.coffee
+30 ./app/views/users/update.coffee
+31 ./app/views/submissions/new.coffee
+31 ./app/views/tags/search.coffee
+32 ./app/views/media/fill_medium_preview.coffee
+32 ./app/views/profile/update.coffee
+32 ./app/views/questions/reassign.coffee
+33 ./app/views/lectures/edit_structures.coffee
+33 ./app/views/media/fill_quizzable_area.coffee
+33 ./app/views/media/publish.coffee
+33 ./app/views/questions/cancel_solution_edit.coffee
+34 ./app/views/questions/set_solution_type.coffee
+35 ./app/assets/javascripts/remarks.coffee
+35 ./app/assets/javascripts/watchlists.coffee
+35 ./app/views/media/fill_quizzable_preview.coffee
+35 ./app/views/sections/display.coffee
+36 ./app/assets/javascripts/announcements.coffee
+36 ./app/views/media/import_script_items.coffee
+37 ./app/views/quizzes/render_vertex_quizzable.coffee
+
+40 ./app/views/referrals/destroy.coffee
+42 ./app/assets/javascripts/geogebra.coffee
+42 ./app/views/clickers/render_clickerizable_actions.coffee
+42 ./app/views/media/fill_reassign_modal.coffee
+42 ./app/views/questions/update.coffee
+44 ./app/assets/javascripts/main.coffee
+44 ./app/views/media/render_medium_actions.coffee
+46 ./app/views/profile/show_accordion.coffee
+50 ./app/assets/javascripts/users.coffee
+51 ./app/views/items/display.coffee
+52 ./app/views/media/search.coffee
+54 ./app/views/media/catalog/import_preview.coffee
+58 ./app/views/media/update.coffee
+61 ./app/assets/javascripts/chapters.coffee
+71 ./app/views/items/destroy.coffee
+77 ./app/assets/javascripts/notifications.coffee
+80 ./app/assets/javascripts/clickers.coffee
+85 ./app/views/tags/update.coffee
+86 ./app/assets/javascripts/talks.coffee
+87 ./app/assets/javascripts/courses.coffee
+87 ./app/assets/javascripts/sections.coffee
+90 ./app/assets/javascripts/referrals.coffee
+91 ./app/assets/javascripts/tex_preview.coffee
+91 ./app/assets/javascripts/vertices.coffee
+91 ./app/views/items/update.coffee
+93 ./app/views/referrals/update.coffee
+97 ./app/assets/javascripts/items.coffee
+
+😭
+112 ./app/assets/javascripts/lessons.coffee
+130 ./app/assets/javascripts/show_clicker.coffee
+192 ./app/views/media/statistics.coffee
+210 ./app/assets/javascripts/tags.coffee
+293 ./app/views/quizzes/proceed.coffee
+352 ./app/assets/javascripts/questions.coffee
+395 ./app/assets/javascripts/quizzes.coffee
+421 ./app/assets/javascripts/media.coffee
+745 ./app/assets/javascripts/upload.coffee
+6689 in total
+
+# What API? Might need more holistic approach (get rid of API entirely)
+3 ./app/assets/javascripts/api/v1/medium.coffee
+3 ./app/assets/javascripts/api/v1/tag.coffee
```

Comment:
Do you have any idea what this API is supposed to be for? There is no route for any API (and any other reference to that in our code base). Probably this is just a remnant of something that existed before and has long been forgotten. Also, these coffescript files do not contain any code. So I would suggets to get rid of them and these lines (now or later).

Reply:
Yeah, I also don't know what this API is used for but didn't check more thoroughly. Would do that later in another PR where I check this in more detail to be sure that we really don't need it.